homeassistant/components/switcher_kis/strings.json

```diff
@@ -1,6 +1,12 @@
 {
   "config": {
     "step": {
+      "user": {
+        "description": "Enter your token here\nDue to security reason, some devices require a token to operate,\nthe token can be acquired from this link: https://switcher.co.il/GetKey/\nNote: minimum firmware version required for the token is 2.31",
```

Comment:
I suggest changing 'Due to security reason' to 'For security reasons'.

Reply:
Done